assessments/FastBridge/seeds/assessmentCategoryDescriptors.csv

```diff
@@ -0,0 +1,2 @@
+codeValue,description,namespace,shortDescription
```

Comment:
this can be included in assessment.csv

Reply:
Removed descriptor seed file and added contents in assessment.csv